Admin/AdminInterface.php

```diff
@@ -266,18 +266,6 @@ public function getParentFieldDescription();
     public function hasParentFieldDescription();
 
     /**
-     * translate a message id.
-     *
-     * @param string $id
-     * @param array  $parameters
-     * @param null   $domain
-     * @param null   $locale
-     *
-     * @return string the translated string
-     */
-    public function trans($id, array $parameters = array(), $domain = null, $locale = null);
-
-    /**
```

Comment:
> it's nearly impossible to have a custom implementation

Ok, so no need to revert. We can consider it isn't a BC-break in this very particular case I think.